tools/coq_makefile.ml

```diff
@@ -151,6 +151,21 @@ let generate_makefile oc conf_file local_file local_late_file dep_file args proj
     ] in
   output_string oc s
 
+let generate_meta_file p =
+  match p.meta_file with
+  | None -> p
+  | Some f ->
+      let ext = Filename.extension f in
+      if ext = ".in" then
+        let meta_file = Filename.chop_extension f in
+        let oc = open_out meta_file in
+        (* META generation is just a renaming for now, we lack some metadata *)
+        output_string oc (read_whole_file f);
```

Comment:
shouldn't this be done in the makefile?

Reply:
coq_makefile already does some substitutions (to CoqMakefile.in) and I thought that maybe one day we could also support some variables to help writing the findlib file. Doing substituons in the Makefiles would still be possible, but less nice IMO.

I can move this code to the CoqMakefile.in if you prefer, no strong opinion.

Reply:
now that there is a real meta generation, this remark is outdated